### PR TITLE
ongoing work.

### PR DESCRIPTION
**Pull request by dave-voltdb**
1. Fix up silly stars printing. It's always annoyed me that the stars had a different width to the text!

2. Define constant for shutdown timeout (easy to change). I'm currently messing around in shutdown so this is convenient. 

3. Distinguish timeout shutdown from intended shutdown path.

4. Sites not executing RealVoltDB.shutdown() should wait forever, not just hope 10 secs is enough.

### CODE REVIEW

File: src/frontend/org/voltcore/utils/CoreUtils.java [hunk 1]
```diff
@@ -1196,43 +1195,45 @@ public static void printAsciiArtLog(VoltLogger vLogger, String msg, Level level)
     {
         if (vLogger == null || msg == null || level == Level.OFF) { return; }
 
-        // 80 stars in a line
+        // Length of stars = msg length, plus 4 for the surrounding "* " and " *"
+        int width = msg.length() + 4;
         StringBuilder starBuilder = new StringBuilder();
-        for (int i = 0; i < 80; i++) {
-            starBuilder.append("*");
+        for (int i = 0; i < width; i++) {
```
Comment (luyangco) [inline src/frontend/org/voltcore/utils/CoreUtils.java:1201]: For aesthetic reasons, you don't want `width` wider than screen width.

File: src/frontend/org/voltdb/sysprocs/Shutdown.java [hunk 2]
```diff
@@ -107,7 +111,9 @@ public void run() {
                     }
                     else {
                         try {
-                            Thread.sleep(10000);
+                            do { // wait here until process exit
```
Comment (luyangco) [inline src/frontend/org/voltdb/sysprocs/Shutdown.java:114]: Alternatively you can designate the site with lowest site id to do the actual shutdown work, rest of the sites don't have to participate. 
```Java
if (context.isLowestSiteId()) {
    // define the shutdown runnable
    // start shutdown thread
}
```

File: src/frontend/org/voltcore/utils/CoreUtils.java (hunk 1 above)
Comment (dave-voltdb) [reply src/frontend/org/voltcore/utils/CoreUtils.java:1201]: If the text is allowed to be that wide why should the stars not be as wide too?

My terminals typically net out at around 120 ...

File: src/frontend/org/voltdb/sysprocs/Shutdown.java (hunk 2 above)
Comment (dave-voltdb) [reply src/frontend/org/voltdb/sysprocs/Shutdown.java:114]: I think the intent was to get the other sites into a known place not executing anything.  But I could be wrong.   My change just keeps them there no matter what.

**Pull request merged**